app/jobs/mailchimp_inviter_job.rb

```diff
@@ -2,15 +2,19 @@ class MailchimpInviterJob < ActiveJob::Base
   queue_as :default
 
   def perform(email:, first_name:, last_name:)
-    gibbon = Gibbon::Request.new(api_key: OperationCode.fetch_secret_with(name: :mailchimp_api_key))
-    list_id = OperationCode.fetch_secret_with name: :mailchimp_list_id
+    begin
```

Comment:
This begin is redundant. You can remove it and place your rescue at the same level as the method definition.